blog/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    password = models.CharField(max_length=128)
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="post_user"
+    )
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now=True)
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(
+        "User", on_delete=models.CASCADE, related_name="comm_user"
```

Comment:
The related_name 'comm_user' might be misleading. It suggests a reverse relation from User to Commentary, but it should be more descriptive, like 'commentaries', to indicate that a user can have multiple commentaries.